### DISCUSSION TITLE
samplerate=48000 liquidsoap

### DISCUSSION THREAD

**Discussion by ghost** [AzuraCast/AzuraCast#5284] [Q&A]
Hello everyone, what request should be made to the user configuration of liquid soap so that it is not 44100 but 48000? Tell me please
this query didn't help.
set("frame.audio.samplerate",48000)
print(get(default=0,"frame.audio.samplerate"))
output.dummy(blank())

**Comment by ghost** [comment]
and tell me what kind of address there should be, information is nowhere to be found, at least an example
![IMG_20220411_171847](https://user-images.githubusercontent.com/102664846/162772437-a32a3ea7-cf7e-47d8-b07b-ddbdb773fe3b.jpg)

**Reply by SpeedyCharly** [reply]
If you leave it blank it will give you a default value, but you can set it in this format ; /[mount_name].mp3 - example; /radio.mp3

**Reply by ghost** [reply]
@SpeedyCharly thank you for your response👍

**Comment by ghost** [comment]
@SpeedyCharly i tried to put it in there /radio22 for example, but I get a 404 error because it opens a link like "server.com/radio22"

**Reply by SpeedyCharly** [reply]
Didn't you add the extension .mp3? That part isn't optional and has to be there unless you stream in another format...

Excerpted from another discussion;

[See this excerpt from the IceCast documentation:

Each Icecast [or Shoutcast] server can house multiple broadcasts (or mountpoints) each containing a separate stream of content. A ‘mountpoint’ is a unique name on your server identifying a particular stream - it looks like a filename, such as ‘/stream.ogg’. A listener can only listen to a single mountpoint at a time. This means you can have a single Icecast server contain either multiple broadcasts with different content, or possibly the same broadcast but with streams of different bitrates or qualities. In this case each broadcast or stream is a separate mountpoint.]

**Comment by ghost** [comment]
@SpeedyCharly 
![IMG_20220414_102708](https://user-images.githubusercontent.com/102664846/163346414-f0d83f9e-f01e-42d4-9657-bcef4b5efca5.jpg)
![IMG_20220414_102550](https://user-images.githubusercontent.com/102664846/163346423-7b523ffa-7ea4-4942-b047-f441b26918f8.jpg)
![IMG_20220414_102244](https://user-images.githubusercontent.com/102664846/163346428-3db9c05e-2723-4be5-a713-6b29332ee16b.jpg)
![IMG_20220414_102135](https://user-images.githubusercontent.com/102664846/163346431-395cd4e9-9107-4366-8ea8-950ab94d2725.jpg)
and if I prescribe even so :8000/superadio.mp3 then the current stream takes the form server.com/radio/8000/radio.mp3 but the rest of the streams remain without a port and a new address /superadio.mp3
